lib/DBDefs/Default.pm

```diff
@@ -271,6 +271,15 @@ sub CACHE_MANAGER_OPTIONS {
     return \%CACHE_MANAGER_OPTIONS
 }
 
+# Sets the TTL for entities stored in memcached, in seconds. On slave servers,
+# this is set to 1 hour by default, to mitigate MBS-8726. On standalone
+# servers, this is set to 0 (meaning no expiration is set), because cache
+# invalidation is already handled by the server in that case.
+sub ENTITY_CACHE_TTL {
+    return 3600 if shift->REPLICATION_TYPE == RT_SLAVE;
+    return 0;
+}
```

Comment:
Cool, that's something we should try, I think. I do like memcached but I'm not sure it has a lot of technical advantages for us over redis. With redis we'd at least have a way to delete keys by prefix using SCAN, AFAICT, and maybe be able to set up local, read-only replicated instances on each app server if the latency isn't too bad.